src/Extension.php

```diff
@@ -205,11 +214,19 @@ public function getConfigKey()
      */
     public function process(ContainerBuilder $container)
     {
-        $passes = $this->getCompilerPasses();
+        //Controller handles the process.
+    }
 
-        foreach ($passes as $pass) {
-            $pass->process($container);
-        }
+    public function loadController(ContainerBuilder $container)
```

Comment:
This has to be defined in `services.xml` (and `services-2.3.xml`) so we do not mix the definitions and they are in one place.